### PR TITLE
Add a new view example for link buttons

### PR DESCRIPTION
**Pull request by Ay-355**
## Summary

This PR adds a new view example showcasing how to create a link button.

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by Rapptz** [comment]
I don't really think this is a good example of a link button. Also, this constructor got simplified so someone doesn't have to do it this way.

**Comment by Ay-355** [comment]
Are there any other examples of a link button that would be a better fit for the repo, or is the example not needed?

File: examples/views/link.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        print('------')
+
+
+# Define a simple View that gives us a google link button.
+# We take in `query` as the query that the command author requests for
+class Google(discord.ui.View):
+    def __init__(self, query: str):
+        super().__init__()
+        # we need to replace spaces with plus signs to make a valid url. Discord will raise an error if it isn't valid.
+        self.query = query.replace(' ', '+')
```
Comment (Rapptz) [inline examples/views/link.py:20]: No, you want to use [`urllib.parse.quote_plus`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote_plus). Be sure to import it at the top as `from urllib.parse import quote_plus`.

```suggestion
        # we need to quote the query string to make a valid url. Discord will raise an error if it isn't valid.
        query = quote_plus(query)
        url = f'https://www.google.com/search?q={query}'
```

File: examples/views/link.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+        # Link buttons cannot be made with the decorator
+        # Therefore we have to manually create one.
+        # We make a valid google search url with the query and add it to the url parameter.
+        # Then we add it to the view
+        self.add_item(
+            discord.ui.Button(style=discord.ButtonStyle.link, label='Click Here', url='https://www.google.com/search?q=' + self.query)
+        )
```
Comment (Rapptz) [inline examples/views/link.py:28]: The construction could be severely simplified.

```suggestion
        self.add_item(discord.ui.Button(label='Click Here', url=url))
```

**Review by Rapptz — CHANGES_REQUESTED** [review]

**Review by Rapptz — APPROVED** [review]
Thanks.

File: examples/views/link.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        self.add_item(discord.ui.Button(label='Click Here', url=url))
+
+
+bot = GoogleBot()
+
+
+@bot.command()
+async def google(ctx: commands.Context, *, query: str):
+    """Returns a google link for a query"""
+    await ctx.send(f"Google Result for: `{query}`", view=Google(query))
```
Comment (Rapptz) [inline examples/views/link.py:36]:
```suggestion
    await ctx.send(f'Google Result for: `{query}`', view=Google(query))
```

**Pull request merged**